src/index.html

```diff
@@ -13,6 +13,54 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
```

Comment:
It would be better to use semantic tags where possible. For example, instead of using a `div` with class `container`, you could use a `section` or `main` tag. This will improve your page SEO and help screen readers. Remember, `div` does not have any meaning.